awscrt/mqtt.py

```diff
@@ -543,10 +543,11 @@ def unsubscribe(self, topic):
         future = Future()
         packet_id = 0
 
-        def unsuback(packet_id):
-            future.set_result(dict(
-                packet_id=packet_id
-            ))
+        def unsuback(packet_id, error_code):
```

Comment:
is this going to cause backwards compatibility problems?

Reply:
These are private callbacks within the implementation, so they shouldn't.

Reply:
Yep, this is fine,
it's just a private function that routes info from the CRT to the actual `future` the user is dealing with